docs/multidict.rst

```diff
@@ -442,3 +442,15 @@ Environment variables
 
       The pure-Python (uncompiled) version is roughly 20-50 times slower than
       its C counterpart, depending on the way it's used.
+
+.. envvar:: MULTIDICT_DEBUG_BUILD
+   
+   An environment variable that instructs the packaging scripts to compile
+   the C-extension based variant of :mod:`multidict` with debug symbols.
+   This is useful for debugging the C-extension code, but it will result in
+   a larger binary size and slower performance.
```

Comment:
Feels like this would be more correct?
```suggestion
   a larger binary size and lower performance.
```

Reply:
adjusted on the other PR in https://github.com/aio-libs/multidict/pull/1146/commits/031ddeb43be1a4968cc4e21be482c8406736b4d3